lace_grid.inx

```diff
@@ -1,34 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
 
-	<_name>Regular Grid</_name>
-	<id>edu.uvic.vmi.mathart.lace_grid</id>
-	
-	<dependency type="extension">org.inkscape.output.svg.inkscape</dependency>
-	<dependency type="executable" location="extensions">lace_grid.py</dependency>
-	<dependency type="executable" location="extensions">inkex.py</dependency>
-	
-	<param name="description" type="description">Creates a printable rectangular grid of dots.</param>
-	
-	<param name="filllabel" type="description">========== Grid style ==========</param>
-	<param name="angle" type="float" precision="2" min="30" max="89" _gui-text="Grid angle (degrees):">45.0</param>
-	<param name="distance" type="float" precision="2" min="1" max="99" _gui-text="Distance between footside pins (mm):">10.0</param>
-	<param name="filllabel" type="description">========== Grid size ==========</param>
-	<param name="width" type="float" precision="2" min="10" max="1000" _gui-text="Patch width (mm):">100</param>
-	<param name="height" type="float" precision="2" min="10" max="1000" _gui-text="Patch height (mm):">100</param>
-	<param name="filllabel" type="description">========== Dot properties ==========</param>
-	<param name="size" type="float" precision="2" min="0.1" max="10" _gui-text="Dot diameter (mm):">0.5</param>
-	<param name="color" type="color" _gui-text="Dot color (#RRGGBBAA):">#000000FF</param>
-	
-	<effect>
-	    <object-type>all</object-type>
-	    <effects-menu>
-	        <submenu _name="Bobbin Lace"/>
-	    </effects-menu>
-	</effect>
-	
-	<script>
-	    <command reldir="extensions" interpreter="python">lace_grid.py</command>
-	</script>
-	
+    <_name>Lace Grid</_name>
+    <id>edu.uvic.vmi.mathart.lace_grid</id>
+    <dependency type="extension">org.inkscape.output.svg.inkscape</dependency>
+    <dependency type="executable" location="extensions">lace_grid.py</dependency>
+    <dependency type="executable" location="extensions">inkex.py</dependency>
+    <dependency type="executable" location="extensions">simplestyle.py</dependency>
+    <dependency type="executable" location="extensions">pturtle.py</dependency>
+    
+    <param name="desc_unit" type="description">Builds a grid of pins.</param>
+    <param name="angle" type="float" precision="2" min="30" max="89" _gui-text="Grid angle (degrees):">45.0</param>
+	<param name="desc_unit" type="description">============ Patch Size ============</param>
+    <param name="units" _gui-text="Measurement units" type="optiongroup" appearance="minimal">
+        <option value="mm">mm</option>
+        <option value="cm">cm</option>
+        <option value="in">in</option>
+        <option value="px">px</option>
+        <option value="pt">pt</option>
+        </param>
+    <param name="desc_unit" type="description">Spacing is vertical dist between footside pins:</param>

```

Comment:
I'm not angry, I'm trying to be polite with please. Of course you lack the knowledge a lace maker has and you can't read my mind. Indeed other non lace makers might not understand it too, but the plugins are grouped under bobbin lace and my current version of the readme refers to the user guide in the wiki-pages. Details belong there, in my opinion. Currently it is a bit of a detour to get at those pages, to fix that is part of #1, assigned to myself.

I intend to try the squash-merge recently implemented by github. In that context I guess its cleaner you undo your misunderstandings yourself and solve merge conflicts.

You removed the clutter but did not move the option to its original group, see the screenshot on my version of the readme.